matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/content/UploadContentWorker.kt

```diff
@@ -200,6 +200,11 @@ internal class UploadContentWorker(val context: Context, params: WorkerParameter
                     Timber.e(failure, "## FileService: Failed to update file cache")
                 }
 
+                // Fix: OpenableColumns.SIZE may return -1
+                if (params.attachment.size <= 0) {
```

Comment:
```suggestion
                if (params.attachment.size < 0) {
```

?

Reply:
I was afraid of any other Android api related bugs. But your suggestion makes sense, done.

Reply:
(reverted to `<=` because sometimes the system returns 0, which is wrong